tools/nl/embeddings/data/preindex/medium/sv_descriptions.csv

```diff
@@ -1686,7 +1686,7 @@ Count_HousingUnit_IncompletePlumbingFacilities_OccupiedHousingUnit,Number of Occ
 Count_HousingUnit_NoCashRent,Number of Renter-Occupied Housing Units with No Cash Rent
 Count_HousingUnit_OccupiedHousingUnit,Number of Occupied Housing Units
 Count_HousingUnit_OwnerOccupied,Number of Owner-Occupied Housing Units
-Count_HousingUnit_RenterOccupied,Number of Renter-Occupied Housing Units
```

Comment:
For future changes, per Bo's observations, I think we should use terminology consistently.  e.g., other occurrences of "Renter-occupied Housing units" should also be "rented houses".  Have a TODO?

Reply:
+1 for TODO. And also there are owner occupied houses we want to clean up too.

Reply:
don't think I can leave comments in csv files, so added to buganizer: https://b.corp.google.com/issues/346618949